supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -164,6 +164,20 @@
 
 *See also*:
 
+[discrete]
+[[organization-administrator]]
+==== image:images/yes.png[yes] Organization Administrator (noun)
+*Description*: From https://access.redhat.com/articles/1757953[Roles and Permissions for Red Hat Customer Portal]: Organization Administrator: This is the highest permission level with full access to content and features. This is the only role that can manage users and control their access and permissions on an account.
```

Comment:
Even though you do provide that link, I'd consider incorporating that it's the highest permission level for Red Hat accounts

```suggestion
*Description*: From https://access.redhat.com/articles/1757953[Roles and Permissions for Red Hat Customer Portal]: Organization Administrator: This is the highest permission level for a Red Hat account with full access to content and features. This is the only role that can manage users and control their access and permissions on an account.
```